src/components/20-molecules/footer-small/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@axa-ch/footer-small",
+  "version": "1.0.0",
```

Comment:
Are we released? is it not better to just release beta till is completly merged?

Reply:
Yes we are already released. :/ Marek wanted to use it and is now doing tests with it.